Countly.js

```diff
@@ -521,7 +521,7 @@ Countly.getDeviceIDType = async function () {
         L.e("getDeviceIDType, unexpected null value from native side");
```

Comment:
It think we can remove this but add a log to default value of intToDeviceID to indicate there was an unexpected value coming from the native side

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

Reply:
Done